lang_GENERIC/analyze/Makefile

```diff
@@ -13,7 +13,8 @@ SRC= \
   Dataflow.ml dataflow_reaching.ml dataflow_liveness.ml \
   IL.ml CFG.ml Meta_IL.ml AST_to_IL.ml CFG_build.ml \
   Dataflow_tainting.ml \
-  Test_analyze_generic.ml
+  Test_analyze_generic.ml \
+  unit_typing_java.ml
```

Comment:
maybe good to follow the new convention I started in this directory to use capitalized name for the filename.
So Unit_typing_java.ml (I know it's not the case everywhere in pfff, but I gradually try to change that).

Reply:
You can also rename it Unit_typing_Generic.ml, because it is supposed to work for both Go and Java. Right now
the test file is for Java, but we should add in the same file a test for Go.